### PR TITLE
[C#] Normalize format of exercise-concepts

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Just some normalization of the formats used

### CODE REVIEW

File: languages/csharp/reference/exercise-concepts/protein-translation.md [hunk 1]
```diff
@@ -15,16 +15,16 @@
 - new keyword: used to instantiate a class
 - lists: a `List<T>` temporarily holds the results before converting them to an array (Add and ToArray methods)
 - for loop: iterate through the strands
-- integers: `int` type used as a counter in the loop 
-- math operations: <, *, and ++
+- integers: `int` type used as a counter in the loop
+- math operations: <, \*, and ++
```
Comment (robkeim) [inline languages/csharp/reference/exercise-concepts/protein-translation.md]: Is this change intended?

Comment (ErikSchierboom) [reply languages/csharp/reference/exercise-concepts/protein-translation.md]: Nope. Will fix
